forc-plugins/forc-debug/docs/walkthrough.md

```diff
@@ -79,46 +79,116 @@ Now we would like to inspect the program while it's running. To do this, we firs
 
 ```json
```

Comment:
Instead of this code snippet, should we just link to the file?

Reply:
Yeah that make sense, the only difference between this and what that doc had is the formatting and addition of `body` key so we can revert the old formatting if this is too long as an alternative